design-documents/graph-ql/coverage/add-items-to-cart-single-mutation.md

```diff
@@ -12,7 +12,124 @@ In order to simplify the flow of adding different type of products to wishlist a
 - Options are sets of predefined values that a buyer can add to cart/wishlist in addition to a product.
 - The difference between wish list and cart payloads is that all fields in wishlist are optional.
 
-### Examples
+### Options 
+
+Each product may have options. Option can be of 2 types (see example below):
+  - `selected_options` - predefined and selected by customer option. E.g. it can be customizable option: `color: green` or bundle option: `Memory: 24M, Warranty: 1y`:
+``` 
+    "selected_options" : [
+         "Y3VzdG9tLW9wdGlvbi8yNC80Mg=="
+     ],
+```
+  - `entered_options` - option entered by customer like: text field, image, etc.
+``` 
+    "entered_options": [
+      {
+        id: "Y3VzdG9tLW9wdGlvbi8zMQ==",
+        value: "Vasia's awesome cup"
+      }
+    ]
+```
+
+We can consider "Selected Option" and "ID for Entered Option" as UUID. They meet the criteria:
+- Represent specific option. 
```

Comment:
"Selected Option" represents option value, while "ID for Entered Option" represents option.

Reply:
done